src/Assembly.js

```diff
@@ -222,6 +223,7 @@ class Assembly extends React.Component {
 
   coordinator_register() {
     this.coordinator_account.persist()
+      .then(() => { this.currentPage = CoordinatorHome })
   }
```

Comment:
This function isn't used any more, so this doesn't make a real difference.
It's nice to note it for consistency though.

Reply:
Gotcha